bin/rustdoc

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+USAGE="Builds Tokio's RustDocs.
+
+USAGE:
+    $(basename "$0") [FLAGS] [<rustdoc-options>...]
+
+FLAGS:
+    -h, --help      Show this help text and exit
+    -u, --unstable  Build documentation for unstable features
+
+NOTES:
+    Additional arguments are passed to the RustDoc invocation. For
+    example:
+
+        $(basename "$0") --open
+
+    will open the documentatiion in a web browser.
+
+    The '-u' or '--unstable' flag will build documentation for unstable
+    features. Unstable features will also be documented if the RUSTFLAGS
+    environment variable contains \"--cfg tokio_unstable\". For example:
+
+        RUSTFLAGS=\"--cfg tokio_unstable\" $(basename "$0")"
+
+set -euo pipefail
+cd "$(dirname "$0")"/..
+
+RUSTDOCFLAGS="${RUSTDOCFLAGS:-} --cfg docsrs"
```

Comment:
```suggestion
RUSTFLAGS="${RUSTFLAGS:-}"
RUSTDOCFLAGS="${RUSTDOCFLAGS:-} --cfg docsrs"
```

Since the `-u` option is set, I think L30 (`echo "${RUSTFLAGS}"`) and L43 (`RUSTFLAGS+=...`) will cause an error if RUSTFLAGS is not set.